scripts/mockgen.sh

```diff
@@ -2,7 +2,7 @@
 
 mockgen_cmd="mockgen"
 $mockgen_cmd -source=client/account_retriever.go -package mock -destination testutil/mock/account_retriever.go
-$mockgen_cmd -package mock -destination store/mock/tendermint_tm_db_DB.go github.com/tendermint/tm-db DB
+$mockgen_cmd -package mock -destination store/mock/cosmos_db_DB.go github.com/cosmos/cosmos-db DB
```

Comment:
Have you ran `make mocks`?
If so I think the previous mock can be deleted.

Reply:
oh wierd -- I thought I had deleted it.  Thanks!

The thing that is requiring so much re: dependencies is the interplay between iavl and tm-db